lib/cylc/mp_pool.py

```diff
@@ -17,16 +17,19 @@
 # along with this program.  If not, see <http://www.gnu.org/licenses/>.
 """Manage queueing and pooling of subprocesses for the suite server program."""
 
-from collections import deque
 import os
 import time
 from pipes import quote
 from signal import SIGKILL
 from subprocess import Popen, PIPE
 from tempfile import TemporaryFile
+from collections import deque
 from threading import RLock
-
 from cylc.cfgspec.glbl_cfg import glbl_cfg
+import time
+import json
+import traceback
+from signal import signal, alarm, SIGALRM
```

Comment:
New imports still used?

Reply:
Yes, I'll remove those.  That was from the original function timeout mechanism, not needed now after your process pool refactor.

Reply:
Done.